source/_docs/installation/synology.markdown

```diff
@@ -175,68 +274,218 @@ case $1 in
         exit 1
         ;;
 esac
-
 ```
 
-Create links to python folders to make things easier in the future:
+## {% linkable_title Controlling Home Assistant %}
 
+* Start Home Assistant:
 ```bash
-# ln -s /volume1/@appstore/py3k/usr/local/bin/python3 python3
-# ln -s /volume1/@appstore/py3k/usr/local/lib/python3.5/site-packages/homeassistant homeassistant
+/volume1/homeassistant/hass-daemon start
 ```
-
-Set the owner and permissions on your config folder
-
+* Stop Home Assistant:
 ```bash
-# chown -R homeassistant:users /volume1/homeassistant
-# chmod -R 664 /volume1/homeassistant
+/volume1/homeassistant/hass-daemon stop
 ```
-
-Make the daemon file executable:
-
+* Restart Home Assistant:
+```bash
+/volume1/homeassistant/hass-daemon restart
+```
+* Upgrade Home Assistant:
 ```bash
-# chmod 755 /volume1/homeassistant/hass-daemon
+# sudo /volume1/@appstore/python3/bin/python3 -m pip install --upgrade homeassistant
 ```
+<p class="note">
+If you need a update for Python 3 or added a component which failed because of a Python module not installing on your Synology NAS with error code "_Failed building wheel for *Module_Name*_", delete the "*~/spksrc*" directory and redo the guide, but add the failed python module to "*requirements.txt*" from "[Preparing compiling environment](#preparing-compiling-environment)".
```

Comment:
If you need a update for Python 3 or added a component which failed because of a Python module not installing on your Synology NAS with error code _Failed building wheel for *Module_Name*_, delete the `~/spksrc` directory and redo the guide, but add the failed python module to `requirements.txt` from [Preparing compiling environment](#preparing-compiling-environment).